src/DataViews/include/DataViews/ModulesDataView.h

```diff
@@ -38,6 +38,13 @@ class ModulesDataView : public DataView {
                  orbit_client_data::ModuleInMemory module_in_memory);
   void UpdateModules(const orbit_client_data::ProcessData* process);
 
+  void OnSelect(const std::vector<int>& rows) override {
```

Comment:
Should this be in `ModulesDataView.cpp`?

Reply:
`OnSelect` is overrided in different ways in sampling data view and live data view.

Reply:
Oh sorry. I think I responded to the wrong comment. I wanted to suggest moving the implementation to the cpp file and not change anything in the inheritance tree.

Reply:
There is no real reason to put this OnSelect anywhere else. Closing this comment

Reply:
Sorry, my mistake. I thought the suggestion was moving it to `DataView`.  🙈  I'm a bit blind...